ci-operator/step-registry/openshift/e2e/aws/cert-rotation/openshift-e2e-aws-cert-rotation-commands.sh

```diff
@@ -93,9 +93,14 @@ run-on-first-master "
 # Pod restart workarounds
 run-on-first-master "
   export KUBECONFIG=${KUBECONFIG_NODE_DIR}/localhost-recovery.kubeconfig
-  # Workaround for https://issues.redhat.com/browse/OCPBUGS-42001
-  # Restart Multus before proceeding
-  oc --request-timeout=5s -n openshift-multus delete pod -l app=multus --force --grace-period=0
+  until oc --request-timeout=5s get nodes; do sleep 10; done | /usr/local/bin/tqdm --desc "Waiting for API server to come up" --null
+  ocp_minor_version=$(oc --request-timeout=5s version -o json | jq -r '.openshiftVersion' | cut -d '.' -f2)
```

Comment:
[periodic-ci-openshift-release-master-nightly-4.18-e2e-aws-ovn-ha-cert-rotation-suspend-30d](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/64962/rehearse-64962-periodic-ci-openshift-release-master-nightly-4.18-e2e-aws-ovn-ha-cert-rotation-suspend-30d/1923025800639025152) run into failure:
```
++ oc --request-timeout=5s version -o json
++ cut -d . -f2
++ jq -r .openshiftVersion
Unable to connect to the server: tls: failed to verify certificate: x509: certificate has expired or is not yet valid: current time 2025-05-15T16:22:41Z is before 2025-06-14T16:16:06Z
/usr/local/bin/timeskew.sh: line 81: ocp_minor_version: unbound variable
```

Reply:
Another job [ci/rehearse/periodic-ci-openshift-release-master-nightly-4.20-e2e-metal-ovn-ha-cert-rotation-shutdown-90d](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/64962/rehearse-64962-periodic-ci-openshift-release-master-nightly-4.20-e2e-metal-ovn-ha-cert-rotation-shutdown-90d/1923025800718716928) failed due to tight gcp resources.

Reply:
Is the time of the client and the apiserver not synchronized?

Reply:
Good point - we should store openshift version before time skew